doc/architecture/decisions/0006-scopes-and-auth.md

```diff
@@ -0,0 +1,58 @@
+# 6. Scopes and auth 
+
+[Next >>](9999-end.md)
+
+
+Date: 2023-01-03
+
+## Status
+
+Draft
+
+## Context
+
+There are several routes we have explored for the moment in regard to Authentication and Authorisation.
```

Comment:
I think I'd also explain that we currently do auth in a mix of places through Cognito and internally, and this is making it relatively tricky to manage

Reply:
This is what we still be doing though? The current functionality and the upcoming plan is the same?